src/main/java/com/hackerrank/weather/controller/WeatherApiRestController.java

```diff
@@ -27,7 +27,7 @@ public WeatherApiRestController(WeatherService weatherService) {
         this.weatherService = weatherService;
     }
 
-    @GetMapping(path = "/today")
+    @GetMapping("/today")
```

Comment:
Please add the rest of the attributes that are available:

@RequestMapping:
- consumes
- produces

Swagger:
@Operation
- Summary
- Description
- Response